### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by jeanbuhendwa**
#### In this pull request, I have : 

- [ ] Added RSpec as a gem to the project.
- [ ] Created a Request spec file for all your controllers.
- [ ] Made sure that for each action to check: If response status was correct, If a correct template was rendered and If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    end
+    it 'should include correct placeholder text.' do
+      get users_path
+      expect(response.body).to include('Users List')
+    end
+    it 'should render correct template' do
+      get users_path
+      expect(response).to render_template('index')
+    end
+  end
```
Comment (gemmen29) [inline spec/requests/users_spec.rb:22]: - [ ] Good job on implementing all `user#index` action tests. Kindly also add all tests for the `user#show` action as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for%20each%20action%20you%20check%3A). All action should be tested.

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'return 200' do
+      expect(response.status).to eq(200)
+    end
+    it 'renders the index template' do
+      expect(response).to render_template('index')
+    end
```
Comment (gemmen29) [inline spec/requests/posts_spec.rb:17]: - [ ] Kindly add a test that checks If the response body includes the correct placeholder text for `post#index` action as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20the%20response%20body%20includes%20correct%20placeholder%20text.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @jeanbuhendwa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      get user_path(user)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should render correct template' do
+      get user_path(user)
+      expect(response).to render_template('show')
+    end
+  end
+end
```
Comment (OyePriscilla) [inline spec/requests/users_spec.rb]: - [ ] Please kindly include the third test case (**`If the response body includes correct placeholder text`**) for the `users#show` method as listed in the requirement. You're almost there! Be encouraged. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @jeanbuhendwa 👍 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
All the written tests passed ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @jeanbuhendwa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented code 👍 
4. All tests are passing 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**